internal/nodes/avi_model_l7_translator.go

```diff
@@ -431,6 +434,25 @@ func (o *AviObjectGraph) BuildPolicyRedirectForVS(vsNode []*AviVsNode, hostname
 
 }
 
+func (o *AviObjectGraph) BuildHeaderRewrite(vsNode []*AviVsNode, gslbHost, localHost, key string) {
+	policyname := lib.GetHeaderRewritePolicy(vsNode[0].Name, localHost)
+	rewriteRule := AviHostHeaderRewrite{
+		SourceHost: gslbHost,
+		TargetHost: localHost,
+	}
+
+	rewritePolicy := &AviHttpPolicySetNode{
+		Tenant:        lib.GetTenant(),
+		Name:          policyname,
+		HeaderReWrite: &rewriteRule,
+	}
+	if policyFound := FindAndReplaceHeaderRewriteHTTPPolicyInModel(vsNode[0], rewritePolicy, gslbHost, key); !policyFound && gslbHost != "" {
```

Comment:
should we check if gslbHost == "" and return before calling FindAndReplaceHeaderRewriteHTTPPolicyInModel ?